gluster/file/cmd/glusterfile-provisioner/glusterfile-provisioner.go

```diff
@@ -926,33 +932,35 @@ func main() {
 		glog.Fatalf("Failed to create config: %v", err)
 	}
 
-	provName := provisionerName
-	provEnvName := os.Getenv(provisionerNameKey)
+	// Detemine provisioner identity. Precedence:
```

Comment:
s/Detemine/Determine

Reply:
You did it too, but I see what you mean. :)